Test/dafny0/DafnyLibClient.dfy

```diff
@@ -1,3 +1,11 @@
+// This test isn't possible with the new UI, because it wants to compile a Dafny library to a .dll
+// and use that as part of another Dafny project. It expects the Dafny source of the library to be included in the .dll
+// as a Source attribute. However, 'dafny build' and `dafny run` always include the Dafny runtime,
```

Comment:
Isn't there an option to exclude the Dafny runtime from the build?

Reply:
The old UI has `/useRuntimeLib` yes, but I don't think it would be good to add that here.

Reply:
The translate command should have the option to include the runtime library -- I'm often annoyed/inconvenienced that it does not by default. In fact I would make it the default in all cases and drop the option.

Reply:
The naming is definitely confusing: `/useRuntimeLib` means "assume we can link to a separate runtime library, so DON'T include that code directly in the output".

> The translate command should have the option to include the runtime library -- I'm often annoyed/inconvenienced that it does not by default. In fact I would make it the default in all cases and drop the option.

We at least need the option to not include the runtime library, because otherwise we can't support separate compilation as in this test case: 
https://github.com/dafny-lang/dafny/tree/master/Test/comp/separate-compilation

Reply:
I don't think discussing `/useRuntimeLib` should be in scope for this PR. I can remove the C# comment that this comment thread resulted from the PR if that makes it easier to merge.

As for the `/useRuntimeLib` discussion, the new UI has a subset of the options it'll eventually need. Whatever is missing, we'll add them when we run into a need for them.

The current state is: `dafny build` always includes the runtime, `dafny translate` does not include the runtime unless `--include-runtime` is specified, since it's meant to produce source code that is built by a native build tool, where the Dafny runtime is specified through configuration so it's only included once even if the build includes multiple Dafny projects.

Reply:
Oh please keep the comment, it's very useful background for the future.

I think the current state is good given `translate` supports the `--include-runtime` option. 

@davidcok Does your workflow just tend to be in two steps, where you `translate` so you can look at the target code, and then build/run it using target language commands? I can sympathize with that, but also think we should bias for end user workflows rather than Dafny contributor workflows. Ideally most end users will have no need to look at the target code and just use `build` and `run`. If they are using `translate` because they are integrating with existing code, hopefully there would be more context to help compile and run the system (e.g. a `*.csproj` file that includes `DafnyRuntime`).